src/StudyViewer/StudyViewer.jsx

```diff
@@ -113,22 +113,21 @@ class StudyViewer extends React.Component {
         <div className='h2-typo study-viewer__title'>
           {studyViewerConfig.title}
         </div>
-        {(datasets.length > 0)
-          ? (
-            <Space className='study-viewer__space' direction='vertical'>
```

Comment:
we can remove `study-viewer__space` from the CSS

Reply:
its still used elsewhere in the component, i dont know what effect it might have on the parts that still use it, doesn't appear to do anything but still cautious about some usecase i am not seeing

Reply:
oh, my bad